config/initializers/high_voltage.rb

```diff
@@ -0,0 +1,8 @@
+HighVoltage.configure do |config|
+  config.layout = false
+
+  config.routes = false
+
+  # app/views/angular_template
+  config.content_path = 'angular_template/'
```

Comment:
I kinda like that .. though, it *might* make people think you need to wrap it in a `<script>` element since that's the only place you'd say "ng-template".